advanced-example/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'java'
+apply plugin: 'kotlin'
```

Comment:
Gdy daję `plugins { id 'kotlin' }` dostaję info, że `plugins` nie może występować przed `buildscript`
A gdy daję po, to jest: 
`Plugin 'kotlin' is already on the script classpath. Plugins on the script classpath cannot be applied in the plugins {} block. Add  "apply plugin: 'kotlin'" to the body of the script to use the plugin.`